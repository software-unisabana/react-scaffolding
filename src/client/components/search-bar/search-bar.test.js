import React from 'react';
import { shallow } from 'enzyme';

import SearchBar from './search-bar.component';

describe('SearchBar Component', () => {
	const wrapper = shallow(<SearchBar />);
	it('expect to render SearchBar Component', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
