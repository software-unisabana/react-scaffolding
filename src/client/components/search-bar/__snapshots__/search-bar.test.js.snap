// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar Component expect to render SearchBar Component 1`] = `ShallowWrapper {}`;
